cabal.project

```diff
@@ -61,8 +61,8 @@ package ouroboros-consensus-cardano
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: 4251c0bb6e4f443f00231d28f5f70d42876da055
-  --sha256: 02a61ymvx054pcdcgvg5qj9kpybiajg993nr22iqiya196jmgciv
+  tag: b364d925e0a72689ecba40dd1f4899f76170b894
+  --sha256: 0igb4gnzlwxy1h40vy5s1aysmaa04wypxn7sn67qy6din7ysmad3
```

Comment:
Sorry, why is this being changed? I always keep this `cabal.project` in sync with the one in `cardano-node`.

If you are fixing `stack.yaml` there should not be any changes in `cabal.project`.

Reply:
Because the newer version of that library breaks the build in stack, so I synced them manually, there should be no significant difference.

Reply:
```
cardano-binary                   > [4 of 8] Compiling Cardano.Binary.ToCBOR                                
cardano-binary                   >                                                                   
cardano-binary                   > /tmp/stack-9255e5bf5797aedb/cardano-binary-1.5.0/src/Cardano/Binary/ToCBOR.hs:47:31: error:
cardano-binary                   >     Module ‘Data.Functor.Foldable’ does not export ‘Fix(..)’      
cardano-binary                   >    |                                                              
cardano-binary                   > 47 | import Data.Functor.Foldable (Fix(..), cata, unfix)          
cardano-binary                   >    |                               ^^^^^^^                        
cardano-binary                   >                                                                   
cardano-binary                   > /tmp/stack-9255e5bf5797aedb/cardano-binary-1.5.0/src/Cardano/Binary/ToCBOR.hs:47:46: error:
cardano-binary                   >     Module ‘Data.Functor.Foldable’ does not export ‘unfix’        
cardano-binary                   >    |                                                              
cardano-binary                   > 47 | import Data.Functor.Foldable (Fix(..), cata, unfix)          
cardano-binary                   >    |                                              ^^^^^           
cardano-binary                   >                                                                   
```

Reply:
This version of the library is actually newer (so the bump makes sense to me):
- `b364d925e0a72689ecba40dd1f4899f76170b894` - 11.01.2021
- `4251c0bb6e4f443f00231d28f5f70d42876da055` - 16.12.2020

Reply:
I already have a branch where I am updating dependencies and I also lock db-sync deps to the same ones as used by node.

By all means, use whatever revision you like in stack but changes to `cabal.project` will only cause me problems.